examples/mesh_calibration_api.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
+
+import cv2
+import numpy as np
+import depthai as dai
+from time import sleep
+import argparse
+from pathlib import Path
+
+"""
+If one or more of the additional depth modes (lrcheck, extended, subpixel)
+are enabled, then:
+ - depth output is FP16. TODO enable U16.
+ - median filtering is disabled on device. TODO enable.
+ - with subpixel, either depth or disparity has valid data.
+
+Otherwise, depth output is U16 (mm) and median is functional.
+But like on Gen1, either depth or disparity has valid data. TODO enable both.
+"""
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-res",
+    "--resolution",
+    type=str,
+    default="720",
+    help="Sets the resolution on mono cameras. Options: 720 | 400",
+)
+parser.add_argument(
+    "-md",
+    "--mesh_dir",
+    type=str,
+    default=None,
+    help="Contains mesh files, if not specified data from EEPROM will be used",
+)
+parser.add_argument(
+    "-static",
+    "--static_frames",
+    default=False,
+    action="store_true",
+    help="Run stereo on static frames passed from host 'dataset' folder",
+)
+args = parser.parse_args()
+
+resolution = (640, 400) if args.resolution == "400" else (1280, 720)
```

Comment:
missing 1280, 800.
And when OAK-D-lite comes into picture we need to add that also..